stream-chat-android/src/main/res/values/attrs.xml

```diff
@@ -107,9 +107,13 @@
             <flag name="bold" value="1"/>
             <flag name="italic" value="2"/>
         </attr>
-        <!--Custom ChannelListLayout-->
         <attr name="streamChannelPreviewLayout" format="reference"/>
     </declare-styleable>
+    <declare-styleable name="ChannelsView">
+    <!--Custom ChannelListLayout-->
+        <!-- Text of the default empty state label -->
```

Comment:
I think we don't need the word `default` in these descriptions, as it's for the customizable attribute

```suggestion
        <!-- Text of the empty state label -->
```